project/Build.scala

```diff
@@ -226,7 +226,7 @@ object BijectionBuild extends Build {
 
   lazy val bijectionUtil = module("util").settings(
     osgiExportAll("com.twitter.bijection.twitter_util"),
-    libraryDependencies += "com.twitter" %% "util-core" % "6.20.0"
+    libraryDependencies += "com.twitter" %% "util-core" % "6.23.0"
```

Comment:
it's not a frivolous bump if that's what you mean, if we want the new Buf apis (Shared/Owned) we need it, but it's also possible to define a bijection for the 6.20 Bufs